pom.xml

```diff
@@ -19,146 +19,53 @@
+        <developerConnection>scm:git:https://github.com/DANS-KNAW/easy-schema</developerConnection>
         <tag>HEAD</tag>
     </scm>
 
-    <dependencies>
-        <dependency>
-            <groupId>com.github.pathikrit</groupId>
-            <artifactId>better-files_2.12</artifactId>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.scalatest</groupId>
-            <artifactId>scalatest_2.12</artifactId>
-            <scope>test</scope>
-        </dependency>
-    </dependencies>
-
-    <repositories>
-        <repository>
-            <id>dans-releases</id>
-            <releases>
-                <enabled>true</enabled>
-            </releases>
-            <snapshots>
-                <enabled>false</enabled>
-            </snapshots>
-            <url>https://maven.dans.knaw.nl/releases/</url>
-        </repository>
-        <repository>
-            <id>dans-snapshots</id>
-            <releases>
-                <enabled>false</enabled>
-            </releases>
-            <snapshots>
-                <enabled>true</enabled>
-            </snapshots>
-            <url>https://maven.dans.knaw.nl/snapshots/</url>
-        </repository>
-    </repositories>
-
-    <build>
-        <resources>
-            <resource>
-                <directory>src/main/resources</directory>
-                <targetPath>${project.build.directory}/version</targetPath>
-                <includes>
-                    <include>version</include>
-                </includes>
-                <filtering>true</filtering>
-            </resource>
-            <resource>
-                <directory>src/main/resources</directory>
-                <excludes>
-                    <exclude>version</exclude>
-                </excludes>
-            </resource>
-        </resources>
+   <build>
         <plugins>
             <plugin>
-                <groupId>com.mycila</groupId>
-                <artifactId>license-maven-plugin</artifactId>
-                <configuration combine.children="append">
-                    <excludes combine.self="override">
-                        <exclude>src/main/resources/extern/*.xsd</exclude>
-                        <exclude>src/main/resources/extern/**/*.xsd</exclude>
-                    </excludes>
-                    <includes>
-                        <include>pom.xml</include>
-                        <include>src/test/scala/**</include>
-                        <include>src/main/resources/**/*.xsd</include>
-                        <include>src/main/resources/**/*.xml</include>
-                    </includes>
+                <artifactId>maven-deploy-plugin</artifactId>
+                <version>3.0.0-M1</version>
+                <configuration>
+                    <skip>true</skip>
+                </configuration>
+            </plugin>
+            <plugin>
```

Comment:
You have declared `maven-release-plugin` twice! Note that Maven warns about this at the start of the build log.